deployment/terraform/resources/cosmosdb.tf

```diff
@@ -100,3 +100,33 @@ resource "azurerm_cosmosdb_sql_container" "items" {
   database_name       = azurerm_cosmosdb_sql_database.pctasks.name
   partition_key_path  = "/stac_id"
 }
+
+resource "azurerm_cosmosdb_sql_container" "create-item-errors" {
+  name                = "create-item-errors"
+  resource_group_name = data.azurerm_cosmosdb_account.pctasks.resource_group_name
+  account_name        = data.azurerm_cosmosdb_account.pctasks.name
+  database_name       = azurerm_cosmosdb_sql_database.pctasks.name
+  partition_key_path  = "/id"
+}
+
+resource "azurerm_cosmosdb_sql_container" "ingest-item-errors" {
+  name                = "create-item-errors"
+  resource_group_name = data.azurerm_cosmosdb_account.pctasks.resource_group_name
+  account_name        = data.azurerm_cosmosdb_account.pctasks.name
+  database_name       = azurerm_cosmosdb_sql_database.pctasks.name
+  partition_key_path  = "/id"
+}
+
+## Cosmos DB Permissions
+
+# The Task Service Principal should be able to write to the `items` container.
```

Comment:
@lossyrob just calling out this (and a couple other) permission changes. This allows the pctasks Task service principal (available to user code) to read / write to *just* the `items` container in Cosmos DB.